.github/pull_request_template.md

```diff
@@ -32,6 +32,7 @@
 
 ## Implementation
 <!--Optional. Add any relevant implementation details that might help the reviewers.-->
+<!--Optional. Add any performance considerations for the reviewers, such as considerations for supporting 1k node clusters.-->
```

Comment:
I wonder if adding this as a comment to this section may just "train" us to overlook/glance over it as the implementation section is rarely filled out. Maybe another optional full section, `## Performance/scaling`, would stand out more?

Reply:
Maybe really add this as a section and just include optional: `## Performance/scaling (optional)`